cmd/goat/util.go

```diff
@@ -42,3 +46,24 @@ func getFileOrStdout(path string) (io.WriteCloser, error) {
 	}
 	return file, nil
 }
+
+func configLogger(cctx *cli.Context, writer io.Writer) *slog.Logger {
```

Comment:
indigo/util/cliutil.SetupSlog()

Reply:
I have a couple things I don't love about `cliutil.SetupSlog`; eg it indirects the env var config out of the `main.go` and `urfav/cli` setup, so the config isn't visible in `--help`.

If I get around to it i'll propose a refactor, or copy this `configLogger` in to a common location.